### DISCUSSION TITLE
[Вопрос] Nopc bot + CMaNGOS World of Warcraft Client Patch 2.4.3

### DISCUSSION THREAD

**Discussion by xBarselonx** [trickerer/Trinity-Bots#234] [Q&A]
trickerer, привет!
Есть в планах сделать NPC bot в проекте CMaNGOS (https://github.com/cmangos) а именно для World of Warcraft TBC 2.4.3 https://github.com/cmangos/mangos-tbc + базы https://github.com/cmangos/tbc-db?

**Comment by trickerer** [comment]
На данный момент нет никаких планов по расширению.

**Comment by xBarselonx** [comment]
trickerer, ты можешь сделать NPC bot в CMaNGOS World of Warcraft Client Patch 2.4.3?

**Reply by trickerer** [reply]
Сделать можно всё, но зачем? Не вижу для этого причин. Зато полно причин этого не делать:
- 2.х.х куда менее популярна чем 3.х.х
- в MaNGOS уже есть родные Playerbots, которых поддерживает сама команда разработчиков эмулятора
- почти не работал с кодом MaNGOS
- нет столько досуга чтобы делать это просто по фану
